src/script/components/content-header.ts

```diff
@@ -132,9 +142,18 @@ export class ContentHeader extends LitElement {
 
         <section>
           <picture>
-            <source srcset="/assets/images/pwab3d.png" media="(max-width: ${BreakpointValues.mediumLower}px)">
-            <source srcset="/assets/images/full_header_logo.png" media="(max-width: ${BreakpointValues.mediumUpper}px)">
-            <img src="/assets/images/pwab3d.png" alt="3d version of the PWABuilder logo">
+            <source
+              srcset="${this.heroImg}"
+              media="(max-width: ${BreakpointValues.mediumLower}px)"
+            />
+            <source
+              srcset="${this.mobileHeroImg}"
+              media="(max-width: ${BreakpointValues.mediumUpper}px)"
+            />
+            <img
+              src="${this.heroImg}"
+              alt="3d version of the PWABuilder logo"
+            />
```

Comment:
Looks like a reformat. I'd like to confirm this is the format our linter is using. @jgw96 @lee-leonardo

Reply:
@lee-leonardo could you confirm if this works or if there's is a better way to this

Reply:
@iharmanpannu I just checked, and yeah, this is how the [ES Prettier extension](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode) formats the code. All good here. 👍